.github/workflows/node-task.yml

```
@@ -0,0 +1,24 @@
+name: Run Node.js Task
+
+on:
+  workflow_call:
+    inputs:
+      task:
+        description: 'Task to execute (typically `yarn run <task>`)'
```

```suggestion
      script:
        description: 'Script to execute (typically `yarn run <script>`)'
```
Wonder if it's helpful to keep terminology consistent with [yarn docs](https://classic.yarnpkg.com/lang/en/docs/cli/run/)?

---

Sounds good 👍